dbt-tests-adapter/dbt/tests/adapter/incremental/test_incremental_microbatch.py

```diff
@@ -63,7 +61,6 @@ def assert_row_count(self, project, relation_name: str, expected_row_count: int)
 
         assert len(result) == expected_row_count, f"{relation_name}:{pformat(result)}"
 
-    @mock.patch.dict(os.environ, {"DBT_EXPERIMENTAL_MICROBATCH": "True"})
```

Comment:
Nice! Let's make sure to apply this change to concrete adapters, I think it's just BigQuery. The existing tests shouldn't break if we left it in, so it's just mostly for cleanliness: 
* https://github.com/dbt-labs/dbt-bigquery/pull/1358/files#diff-933dcacdedf65d9337c2fd85863166cc878c4c8351b07a16d778bb99abaaaa95L33

(feel free to start a fresh dbt-bigquery PR if you'd like!)